serverless/knative_eventing/serverless-kn-trigger.adoc

```diff
@@ -0,0 +1,67 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-kn-trigger"]
+= Using triggers with Knative Eventing
```

Comment:
Should Eventing be capitalized?

Reply:
I would say so, the same for Serving, since these are specific components of Serverless and are generally always capitalized in upstream docs? https://knative.dev/docs/